VirtualList/VirtualList.js

```diff
@@ -889,11 +894,16 @@ VirtualGridList.propTypes = /** @lends sandstone/VirtualList.VirtualGridList.pro
 	snapToCenter: PropTypes.bool,
 
 	/**
-	 * Spotlight Id.
-	 *
-	 * @type {String}
-	 * @private
-	 */
+	* The container id for [Spotlight container]@link spotlight/SpotlightContainerDecorator/#SpotlightContainerDecorator.spotlightId}.
+	*
+	* It is required to restore focus after `VirtualList` remount.
+	*
+	* For example, when the `VirtualList` is within a [Panel]{@link sandstone/Panels.Panel},,
+	* the `Spotlight` will store the id to restore focus when returning to the `Panel`.
```

Comment:
the `Spotlight` will store the `spotlightId` to restore focus when returning to the `Panel`.

Reply:
I fixed